src/utils/api.ts

```diff
@@ -18,37 +18,43 @@ const getBaseUrl = () => {
   return `http://localhost:${process.env.PORT ?? 3000}`; // dev SSR should use localhost
```

Comment:
Tutaj też został użyty process.env zamiast importu `import { env } from "@/env.mjs";`

Reply:
bardzo ładnie wyłapane, to akurat był wcześniej zrobiony boilerplate, ale nie zauważyłem że używali to process.env'a, już poprawiam

Reply:
okej, jednak musi być process.env, inaczej wszystko się psuje